offline.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * The PWA Offline template file
+ *
+ * @package Newspack
+ */
+
+get_header();
+?>
+	<section id="primary" class="content-area">
+		<header class="entry-header">
+			<h1 class="entry-title"><?php esc_html_e( 'Oops! It looks like you&rsquo;re offline.', 'newspack' ); ?></h1>
```

Comment:
Technically the server could also be offline. See https://github.com/xwp/pwa-wp/pull/190

So maybe just “Offline” would be better.

```suggestion
			<h1 class="entry-title"><?php esc_html_e( 'Offline', 'newspack' ); ?></h1>
```

Reply:
Ah, good call! This has been updated in f92f9c1.